### DISCUSSION TITLE
Update Option pickers between screens

### DISCUSSION THREAD

**Discussion by vaskoZGU** [Budibase/budibase#12445] [Help] [answered]
Hi there,

Two screens.
On change, the Option picker in screen A updates the Option picker in screen B

Screen A
![image](https://github.com/Budibase/budibase/assets/117993754/e8f2b9eb-966d-41a7-957f-5b23a7df96fe)

Option picker "doctors" have selected the value "Test"

Screen B
![image](https://github.com/Budibase/budibase/assets/117993754/05654649-90a4-4900-9b1c-37280c6bff47)

The OP in screen B has a value updated from OP in screen A.

Is this possible?

**Comment by melohagan** [comment]
Hey @vaskoZGU 

If you are using different screens, then you may want to look at using [URL Variables](https://docs.budibase.com/docs/url-parameters) and combine that with the *Default Value* property of the fields, e.g. `{{ URL.my_var }}`

**Reply by vaskoZGU** [reply]
Hi @melohagan 
I read the article for URL Variables, but I can't figure it out.
Can you give me some short examples?

Thank you.

**Reply by melohagan** [reply, marked as answer]
A community member wrote up this handy tutorial! https://github.com/Budibase/budibase/discussions/10943

**Reply by vaskoZGU** [reply]
Hi @melohagan,

Thank you. This tutorial helped me to understand URL variables.

Cheers
